data/map planets.txt

```diff
@@ -2725,7 +2725,7 @@ planet "New Tortuga"
 	"required reputation" -10
 	bribe 0.03
 	security 0
-	tribute 6000
+	tribute 1000
```

Comment:
```suggestion
	tribute 1500
```


Based on the arguments presented, I'd say somewhere in the 1500-2000 range would be good.

Reply:
I suppose I could see 1500, but I'm leaning towards 1400. I feel that's as high as I'd go here.

Reply:
Given how extreme 6000 is, I'd really appreciate someone digging up why it was set that high to begin with. I doubt we accidentally set its tribute to double the value of the Syndicate capital. It's quite possible that the Syndicate capital is on the low side; but also just as possible that Tortuga is the secret home of the off-shore bank accounts that will pay generously to keep people from sticking their noses where they don't belong.

I'd like to hear the thoughts of the people working on the pirate campaign/rework here. Possibly the Syndicate people too.

This just seems too obvious to be just a mistake. Although admittedly, it is possible.

Reply:
This is a planet that was added to the game through Exile content, so it was either done by Raven or Lake. I'll request them.

Reply:
It's been at 6000 since the first commit that added it to that branch. Never edited.

Reply:
It was set by Nomadic Volcano